sootup.core/src/main/java/sootup/core/model/Body.java

```diff
@@ -459,12 +459,18 @@ public String toString() {
       }
     }
 
-    public void removeDefLocalsOf(@Nonnull Stmt stmt) {
+    public void removeDefLocalsOf(@Nonnull Stmt stmt, BodyBuilder builder) {
```

Comment:
We do not need the `BodyBuilder` parameter here. We can call `getStmts()` on the object itself e.g. `this.graph.getStmts()`